qtpy/QtDBus.py

```diff
@@ -4,16 +4,30 @@
 # Licensed under the terms of the MIT License
 # (see LICENSE.txt for details)
 # -----------------------------------------------------------------------------
+
 """Provides QtDBus classes and functions."""
 
-# Local imports
-from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PythonQtError
+import sys
```

Comment:
Please add a blank line below this one. We accustom to leave an empty space between standard library and local imports.